CHANGELOG.md

```diff
@@ -3,27 +3,34 @@ Changes
 
 21.05 to 21.09
 --------------
+
 * [Developer]: Added pipeline plugin path configuration option
 * [UI]: Update of Project Activities page to use Ant Design.
 * [UI]: Fixes bug where the page title on Remote IRIDA Connections page was not displaying properly.
-* [UI]: Fixed bug where a remote project in `UNSYNCHRONIZED` state would have the Sync buttons disabled on the Project Synchronization Settings page. (21.05.1)
+* [UI]: Fixed bug where a remote project in `UNSYNCHRONIZED` state would have the Sync buttons disabled on the Project
+  Synchronization Settings page. (21.05.1)
 * [UI]: Added back `Users` option for managers in main navigation. (21.05.1)
+* [Documentation]: Added Swagger and OpenAPI to the IRIDA REST API documentation.
```

Comment:
Can you undo the big formatting changes to this file and just add your line?  It'll end up causing merge conflicts down the line.

Reply:
Please see eca515d78511d9225c8eea4a28188c749ac7ad53 and 151282d5987a051c16b12e1bacb812a36466c19a.